pkg/cmd/server/bootstrappolicy/constants.go

```diff
@@ -96,7 +96,7 @@ const (
 	SDNManagerRoleName        = "system:sdn-manager"
 	OAuthTokenDeleterRoleName = "system:oauth-token-deleter"
 	WebHooksRoleName          = "system:webhook"
-	DiscoveryRoleName         = "system:discovery"
+	DiscoveryRoleName         = "system:openshift:discovery"
```

Comment:
Drop the duplicate rules from this cluster role?

Reply:
> Drop the duplicate rules from this cluster role?

That would affect the scope authorizer amongst other spots.  This solves 90% of our problem.  I'd live with extra rules we can transition at some later point.